docker/authn/docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
   nessie:
     image: ghcr.io/projectnessie/nessie:latest
     ports:
-      - 19120:19120
+      - "19120:19120"
```

Comment:
Should the changes in this file be on `main`?

Reply:
Yes I will cherry-pick these changes and create a PR.